config/initializers/holidays.rb

```diff
@@ -0,0 +1,51 @@
+Rails.application.config.holidays = [
+  Date.new(2012, 1, 2),
+  Date.new(2012, 4, 6),
+  Date.new(2012, 4, 9),
+  Date.new(2012, 5, 7),
+  Date.new(2012, 6, 4),
+  Date.new(2012, 6, 5),
+  Date.new(2012, 8, 27),
+  Date.new(2012, 12, 25),
+  Date.new(2012, 12, 26),
+  Date.new(2013, 1, 1),
+  Date.new(2013, 3, 29),
+  Date.new(2013, 4, 1),
+  Date.new(2013, 5, 6),
+  Date.new(2013, 5, 27),
+  Date.new(2013, 8, 26),
+  Date.new(2013, 12, 25),
+  Date.new(2013, 12, 26),
+  Date.new(2014, 1, 1),
+  Date.new(2014, 4, 18),
+  Date.new(2014, 4, 21),
+  Date.new(2014, 5, 5),
+  Date.new(2014, 5, 26),
+  Date.new(2014, 8, 25),
+  Date.new(2014, 12, 25),
+  Date.new(2014, 12, 26),
+  Date.new(2015, 1, 1),
+  Date.new(2015, 4, 3),
+  Date.new(2015, 4, 6),
+  Date.new(2015, 5, 4),
+  Date.new(2015, 5, 25),
+  Date.new(2015, 8, 31),
+  Date.new(2015, 12, 25),
+  Date.new(2015, 12, 28),
+  Date.new(2016, 1, 1),
+  Date.new(2016, 3, 25),
+  Date.new(2016, 3, 28),
+  Date.new(2016, 5, 2),
+  Date.new(2016, 5, 30),
+  Date.new(2016, 8, 29),
+  Date.new(2016, 12, 26),
+  Date.new(2016, 12, 27),
+  Date.new(2017, 1, 2),
+  Date.new(2017, 4, 14),
+  Date.new(2017, 4, 17),
+  Date.new(2017, 5, 1),
+  Date.new(2017, 5, 29),
+  Date.new(2017, 8, 28),
+  Date.new(2017, 12, 25),
+  Date.new(2017, 12, 26)
+]
```

Comment:
… which I see @threedaymonk replicates below, but instead of maintaining a copy of the JSON, we have a direct mapping to this ruby file. I'm not entirely averse to this, then.